prefq/static/web_interface.js

```diff
@@ -5,91 +5,95 @@ var on_right_preferred      = document.getElementById('right_preferred')
 var left_video              = document.getElementById('left_video')
```

Comment:
Are the changes to this file necessary? If so, why?

Reply:
Looks like the changes are mostly reformatting. Would be nicer to have them in a separate commit, but they don't hurt so it think we can go ahead and merge this.